lnrpc/chainrpc/chainkit.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package chainrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/chainrpc";
+
+// ChainKit is a service that can be used to get information from the
+// chain backend.
+service ChainKit {
+    /* lncli: `chain getblock`
+    GetBlock returns a block given the corresponding block hash.
+    */
+    rpc GetBlock (GetBlockRequest) returns (GetBlockResponse);
+
+    /* lncli: `chain getbestblock`
+    GetBestBlock returns the block hash and current height from the valid
+    most-work chain.
+    */
+    rpc GetBestBlock (GetBestBlockRequest) returns (GetBestBlockResponse);
+
+    /* lncli: `chain getbestblock`
+    GetBlockHash returns the hash of the block in the best blockchain
+    at the given height.
+    */
+    rpc GetBlockHash (GetBlockHashRequest) returns (GetBlockHashResponse);
+}
+
+message GetBlockRequest {
+    // The hash of the requested block.
+    bytes block_hash = 1;
+}
+
+message GetBlockResponse {
+    // The raw bytes of the requested block.
+    bytes raw_block = 1;
```

Comment:
I am not sure if it makes sense to add some filelds to have a more "digested" version while leaving this one in case the client wants to decode the raw block himself.

The server could reverse the hash so we can directly print it too. The neutrino response can be a source of inspiration in case we are following this route.

https://github.com/lightningnetwork/lnd/blob/master/lnrpc/neutrinorpc/neutrino_server.go#L406

Reply:
Following on from our discussion during standup, I've added a TODO here: https://github.com/lightningnetwork/lnd/compare/ab016303a94ad58342efb91c8588cdd9239fbe2a..2d9add6a520bc280052080163eea585379d8daa7#diff-02c2ffdf3e4eeaf8ad78a9b793b6cc1ca9713005c5548ad3aeb16ddb5f939dd8R33-R35